typescript/web/src/components/workspaces/update-plan-banner.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, chakra, Flex, Text } from "@chakra-ui/react";
+import { useRouter } from "next/router";
+import { RiInformationFill } from "react-icons/ri";
+import { useWorkspace } from "../../hooks/use-user";
+
+const InfoIcon = chakra(RiInformationFill);
+export const UpdatePlanBanner = () => {
+  const { slug: workspaceSlug } = useWorkspace();
+  const router = useRouter();
+  const redirectToSettings = () => {
+    router.push(`/${workspaceSlug}/settings`);
+  };
+  return (
+    <Flex
+      direction={{ base: "column", md: "row" }}
+      justify="center"
+      align="center"
+      bg="#EFAB22"
+      zIndex="100"
+      p="4"
+      gridGap="4"
+    >
+      <InfoIcon h="24px" w="24px" />
+      <Text fontWeight="500" textAlign="center">
+        Your trial has expired. Update your payment info to not lose access to
+        your datasets or downgrade to the community plan.
```

Comment:
@nicOwlas I'm not that an expert in English but isn't there a grammar problem with `to not lose access to`? Shouldn't it be `to don't lose access to`?

Anyway what about `to keep access to`

Reply:
`To not lose access to` is grammatically correct but I agree that the double negation should be avoided. Ok to replace with `to keep access to`